src/nifake/tests/test_session.py

```diff
@@ -328,3 +328,34 @@ def test_multiple_return_params(self):
             assert self.patched_errors.handle_error.call_count == 2
             self.patched_errors.handle_error.assert_called_with(session, self.patched_library.niFake_GetCalDateAndTime.return_value)
     '''
+
+    def test_init_with_options_nondefault(self):
+        session = nifake.Session('FakeDevice', 1, True, 'Some string')
+        assert(session.vi == SESSION_NUM_FOR_TEST)
+        self.patched_library.niFake_InitWithOptions.assert_called_once_with(b'FakeDevice', 1, True, ANY, ANY)
+
+    def test_read(self):
+        test_maximum_time = 1000
+        test_reading = 5
+        self.patched_library.niFake_Read.side_effect = self.side_effects_helper.niFake_Read
+        self.side_effects_helper['Read']['reading'] = test_reading
+        with nifake.Session('dev1') as session:
+            session.read(test_maximum_time)
+            from mock import call
+            calls = [call(SESSION_NUM_FOR_TEST, test_maximum_time, ANY)]
+            self.patched_library.niFake_Read.assert_has_calls(calls)
+            assert self.patched_library.niFake_Read.call_count == 1
+
+    def test_read_multipoint(self):
+        test_maximum_time = 1000
+        test_reading_array = 1.2
```

Comment:
shouldn't this be a list?

Reply:
I initially tried it as a list, but I get a typeError saying it should be a float.
In session.py, it is mentioned that this is a float.

Should it actually be a list instead?